README.markdown

```diff
@@ -1512,13 +1512,17 @@ This hook is often used to create per-worker reoccurring timers (via the [ngx.ti
                  return
              end
          end
+
+         -- do something
```

Comment:
I think `do something` is a bit simple here, we can describe it more clear, like:
`do something in timer`.

Reply:
update now.